docs/core/toc.yml

```diff
@@ -2,6 +2,44 @@
   href: index.md
 - name: About .NET Core
   href: about.md
+- name: Install
```

Comment:
Do we need to add breadcrumbs for Install and Linux package managers?

Reply:
yes for Install

Reply:
How do I do that for the package managers? They aren't in their own folder because it's too many folders on the URL.